### PR TITLE
feat: React-Typescript-101.003 EsBuild + TS + React

### PR DESCRIPTION
**Pull request by ehsankhfr**

### CODE REVIEW

File: sessions/React-Typescript-100/101/101.003/README.md [hunk 1]
```diff
@@ -1 +1,182 @@
...
+          import React from 'react';
+          import ReactDOM from 'react-dom';
+          import App from './App';
+     
+          ReactDOM.render(<App />, document.getElementById('app'));
+  
+  11. EsBuild is the module bundler which will compile all the javascript files or modules into single file called as bundle.
+         - Use terminal: `yarn add esbuild esbuild-plugin-inline-image esbuild-css-modules-plugin`
+         - Use terminal: `yarn add -D live-server`
+         - Create file `serve.js`
```
Comment (DJWaddsIW) [inline sessions/React-Typescript-100/101/101.003/README.md:71]: Is this serve or server?

Comment (DJWaddsIW) [inline sessions/React-Typescript-100/101/101.003/README.md:71]: Is this serve or server?

File: sessions/React-Typescript-100/101/101.003/README.md [hunk 2]
```diff
@@ -1 +1,182 @@
...
+
+               const cssModulesPlugin = require("esbuild-css-modules-plugin");
+               const inlineImage = require("esbuild-plugin-inline-image");
+               const {build} = require("esbuild")
+               const liveServer = require("live-server")
+    
+               ;(async () => {
```
Comment (DJWaddsIW) [inline sessions/React-Typescript-100/101/101.003/README.md:78]: Is the ; correct here

Comment (DJWaddsIW) [inline sessions/React-Typescript-100/101/101.003/README.md:78]: Is the ; correct here

**Review by DJWaddsIW — APPROVED** [review]
Just a few questions, but looking good

File: sessions/React-Typescript-100/101/101.003/README.md (hunk 2 above)
Comment (ehsankhfr) [reply sessions/React-Typescript-100/101/101.003/README.md:78]: Yeah, it's is commonly used in JS and is borrowed from early days of JS. A good article to read if you like: https://medium.com/@maxbendick/semicolons-go-at-the-beginning-of-the-line-34ea9d26393a

File: sessions/React-Typescript-100/101/101.003/README.md (hunk 1 above)
Comment (ehsankhfr) [reply sessions/React-Typescript-100/101/101.003/README.md:71]: It's `serve`, and is about serving the dev-server. I just changed it into `dev.js`

File: sessions/React-Typescript-100/101/101.003/README.md (hunk 2 above)
Comment (ehsankhfr) [reply sessions/React-Typescript-100/101/101.003/README.md]: Changed the style of script

**Pull request merged**